frontend/src/components/auth/Login.tsx

```diff
@@ -68,7 +68,10 @@ const Login = (): React.ReactElement => {
           <Text textStyle="body" color="gray.700">
             Enter your credentials to access your account.
           </Text>
-          <FormControl mt="1rem">
+          <FormControl
+            mt="1rem"
+            onKeyPress={(e) => e.key === "Enter" && onLogInClick()}
```

Comment:
This syntax might be a bit confusing because `onLogInClick()` doesn't return a boolean. Consider doing something like 
```typescript
if (e.key === "Enter") {
  onLogInClick();
}
```

Reply:
Just to add to my comment - you can create a custom function to achieve that functionality defined before the JSX component is returned. Similar to how clicks are handled.